tests/dataimportcron_test.go

```diff
@@ -95,6 +95,29 @@ var _ = Describe("DataImportCron", Serial, func() {
 				Satisfy(meta.IsNoMatchError),
 			))
 		}
+
+		By("[AfterEach] Delete the DataImportCron under test")
+		// Delete the DataImportCron under test
+		_ = f.CdiClient.CdiV1beta1().DataImportCrons(ns).Delete(context.TODO(), cronName, metav1.DeleteOptions{})
+		Eventually(func() bool {
+			_, err := f.CdiClient.CdiV1beta1().DataImportCrons(ns).Get(context.TODO(), cronName, metav1.GetOptions{})
+			return errors.IsNotFound(err)
+		}, dataImportCronTimeout, pollingInterval).Should(BeTrue(), "DataImportCron was not deleted")
+
+		By("[AfterEach] Wait for all DataImportCrons UpToDate")
+		// Wait for all DataImportCrons to converge
+		dataImportCrons := &cdiv1.DataImportCronList{}
+		err = f.CrClient.List(context.TODO(), dataImportCrons, &client.ListOptions{Namespace: metav1.NamespaceAll})
+		Expect(err).ToNot(HaveOccurred())
+		for i := range dataImportCrons.Items {
+			cronItem := &dataImportCrons.Items[i]
+			Eventually(func() bool {
```

Comment:
I meant this eventually loop isn't doing any polling since it always loops over the same `dataImportCrons.Items` list

Reply:
So I delete the DataImportCron cronName (created by the tests), then Eventually waits for it to be deleted

Then I get the list of all, loop over it, get each by `cronItem := &dataImportCrons.Items[i]`, and then Eventually waits for the `cronItem`'s conditions (at least that's what I want)

Reply:
yeah but you only fetch the list from the cluster once. so if any cron takes more time to converge, the eventually loop will not catch that, and just time out on a stale entry

Reply:
Oh I seeee! `cronItem` is not really a reference, so we are not really polling the live object... let me try a new patch :) 

Should we also increase a timeout? Maybe 10 minutes? I'm afraid that 4 minutes may not be enough when we have a slow cluster and 6 concurrent imports.

Also changed a loop, using `for _, cronItem := range dataImportCrons.Items`